integration_tests/tests/in_cluster.rs

```diff
@@ -101,15 +100,23 @@ impl InClusterJanusPair {
         // so they need the in-cluster DNS name of the other aggregator, and they can use well-known
         // service port numbers.
 
-        let divviup_api = DivviupApiClient::new(
-            cluster
-                .forward_port(&divviup_api_namespace, "divviup-api", 80)
-                .await,
+        let port = cluster
+            .forward_port(&divviup_api_namespace, "divviup-api", 80)
+            .await
+            .local_port();
```

Comment:
The struct returned by `Cluster::forward_port()` needs to be stored in a separate binding, as it acts as a drop guard for shutting down the port forwarding.